website/pages/use-cases/non-containerized-application-orchestration.jsx

```diff
@@ -0,0 +1,128 @@
+import UseCasesLayout from '../../layouts/use-cases'
+import TextSplitWithImage from '@hashicorp/react-text-split-with-image'
+import FeaturedSliderSection from '../../components/featured-slider-section'
+
+export default function NonContainerizedApplicationOrchestrationPage() {
+  return (
+    <UseCasesLayout
+      title="Non-Containerized Application Orchestration"
+      description="Nomad's flexible workload support enables an organization to run containerized, non containerized, and batch applications through a single workflow. Nomad brings core orchestration benefits to legacy applications without needing to containerize via pluggable task drivers."
+    >
+      <TextSplitWithImage
+        textSplit={{
+          heading: 'Non-Containerized Orchestration',
+          content:
+            'Deploy, manage, and scale your non-containerized applications using the Java, Qemu, or exec drivers.',
+          textSide: 'right',
+          links: [
+            {
+              text: 'Watch the Webinar',
+              url:
+                'https://www.hashicorp.com/resources/move-your-vmware-workloads-nomad',
+              type: 'outbound'
+            }
+          ]
+        }}
+        image={{
+          url: require('./img/non-containerized-orchestration.png'),
+          alt: 'Non-Containerized Orchestration'
+        }}
+      />
+
+      <TextSplitWithImage
+        textSplit={{
+          heading: 'Improve Resource Utilization with Bin Packing',
+          content:
+            'Improve resource utilization and reduce costs for non-containerized applications through Nomad’s bin-packing placements.',
+          textSide: 'left'
+        }}
+        image={{
+          url: require('./img/bin-packing.svg'),
+          alt: 'Bin Packing'
+        }}
+      />
+
+      <TextSplitWithImage
+        textSplit={{
+          heading: 'Zero Downtime Deployments',
+          content:
+            'Apply modern upgrade strategies for legacy applications through rolling updates, blue/green, or canary deployment strategies.',
+          textSide: 'right',
+          links: [
+            {
+              text: 'Read more',
+              url: '/guides/operating-a-job/update-strategies',
+              type: 'inbound'
+            }
+          ]
+        }}
+        image={{
+          url: require('./img/zero-downtime-deployments.png'),
+          alt: ''
+        }}
+      />
+
+      <TextSplitWithImage
+        textSplit={{
+          heading: 'Run on-premise with ease',
```

Comment:
```suggestion
          heading: 'Run On-Premise with Ease',
```

Match headline caps style